backend/src/schema/resolvers/users.js

```diff
@@ -11,5 +11,27 @@ export default {
       params = await fileUpload(params, { file: 'avatarUpload', url: 'avatar' })
       return neo4jgraphql(object, params, context, resolveInfo, false)
     },
+    DeleteUser: async (object, params, context, resolveInfo) => {
+      const { resource } = params
+      const session = context.driver.session()
+
+      if (resource && resource.length) {
+        await Promise.all(
+          resource.map(async node => {
+            await session.run(
+              `
+            MATCH (resource:${node})<-[:WROTE]-(author:User {id: $userId})
```

Comment:
A couple of thoughts here: First of all, I believe, you can do this in one statement. I would try sth. like:
```
MATCH (author:User {id: $userId}), (post:Post)<-[:WROTE]-(author), (comment:Comment)<[:WROTE]-(author)
DETACH DELETE author, post, comment
```
I don't know if it works like that but I guess you get the point.

Reply:
the point is they wouldn't have the choice of deleting just their account, or just their account and posts, just their account and comments, etc...

Reply:
Oh, right, true! We cannot customize `neo4j-graphql-js`.